### PR TITLE
[zh] Update all md documents in the kubeadm

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md [hunk 1]
```diff
@@ -56,7 +56,13 @@ etcd cluster of three members that can be used by kubeadm during cluster creatio
 -->
 * 每个主机必须安装 systemd 和 bash 兼容的 shell。
 * 每台主机必须[安装有容器运行时、kubelet 和 kubeadm](/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm/)。
+<!--
+* Each host should have access to the Kubernetes container image registry (`registry.k8s.io`) or list/pull the required etcd image using
+`kubeadm config images list/pull`. This guide will setup etcd instances as
+[static pods](/docs/tasks/configure-pod-container/static-pod/) managed by a kubelet.
 
+-->
+* 每个主机都应该能够访问 Kubernetes 容器镜像注册表 (registry.k8s.io)，或者使用或者使用 kubeadm config 镜像列表/拖动列出/拖动所需的 etcd 镜像。本指南将把 etcd 实例设置为由 kubelet 管理的[静态 pods](/zh-cn/docs/tasks/configure-pod-container/static-pod/)。
```
Comment (windsonsea) [inline content/zh-cn/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md:65]:
```suggestion
* 每个主机都应该能够访问 Kubernetes 容器镜像仓库 (registry.k8s.io)，
  或者使用 `kubeadm config images list/pull` 列出/拉取所需的 etcd 镜像。
  本指南将把 etcd 实例设置为由 kubelet 管理的[静态 Pod](/zh-cn/docs/tasks/configure-pod-container/static-pod/)。
```

**Review by windsonsea — COMMENTED** [review]
Add a newline char for a long clause. Refer to https://kubernetes.io/zh-cn/docs/contribute/localization_zh/

**Comment by gaogao101** [comment]
@windsonsea Thank you for your advice. I will finish the modification and re-upload as soon as possible. This is my first time to contribute to this community, so I am not familiar with the localization requirements. I have read the document you gave me.

**Comment by gaogao101** [comment]
@windsonsea In addition, I would like to ask if there is anything else I need to criticize

**Comment by mengjiao-liu** [comment]
Have you fetched the latest main branch? I checked the synchronized files, you seem to have not synchronized the content of the following two files? Please check it again.
```diff
$ ./scripts/lsync.sh content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md
diff --git a/content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md b/content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md
index e4214eaf71..9dd8653210 100644
--- a/content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md
+++ b/content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md
@@ -245,8 +245,8 @@ Install `kubeadm`, `kubelet`, `kubectl` and add a `kubelet` systemd service:
 RELEASE="$(curl -sSL https://dl.k8s.io/release/stable.txt)"
 ARCH="amd64"
 cd $DOWNLOAD_DIR
-sudo curl -L --remote-name-all https://storage.googleapis.com/kubernetes-release/release/${RELEASE}/bin/linux/${ARCH}/{kubeadm,kubelet,kubectl}
-sudo chmod +x {kubeadm,kubelet,kubectl}
+sudo curl -L --remote-name-all https://dl.k8s.io/release/${RELEASE}/bin/linux/${ARCH}/{kubeadm,kubelet}
+sudo chmod +x {kubeadm,kubelet}
 
 RELEASE_VERSION="v0.4.0"
 curl -sSL "https://raw.githubusercontent.com/kubernetes/release/${RELEASE_VERSION}/cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service" | sed "s:/usr/bin:${DOWNLOAD_DIR}:g" | sudo tee /etc/systemd/system/kubelet.service
@@ -254,6 +254,8 @@ sudo mkdir -p /etc/systemd/system/kubelet.service.d
 curl -sSL "https://raw.githubusercontent.com/kubernetes/release/${RELEASE_VERSION}/cmd/kubepkg/templates/latest/deb/kubeadm/10-kubeadm.conf" | sed "s:/usr/bin:${DOWNLOAD_DIR}:g" | sudo tee /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
 \```
 
+Install `kubectl` by following the instructions on [Install Tools page](/docs/tasks/tools/#kubectl).
+
 Enable and start `kubelet`:

 ```

```diff
$ ./scripts/lsync.sh content/zh-cn/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md
diff --git a/content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md b/content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md
index c5abcde425..4fa48c4bfb 100644
--- a/content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md
+++ b/content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md
@@ -31,7 +31,7 @@ etcd cluster of three members that can be used by kubeadm during cluster creatio
   the kubeadm config file.
 * Each host must have systemd and a bash compatible shell installed.
 * Each host must [have a container runtime, kubelet, and kubeadm installed](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
-* Each host should have access to the Kubernetes container image registry (`k8s.gcr.io`) or list/pull the required etcd image using
+* Each host should have access to the Kubernetes container image registry (`registry.k8s.io`) or list/pull the required etcd image using
 `kubeadm config images list/pull`. This guide will setup etcd instances as
 [static pods](/docs/tasks/configure-pod-container/static-pod/) managed by a kubelet.
 * Some infrastructure to copy files between hosts. For example `ssh` and `scp`
@@ -276,7 +276,7 @@ on Kubernetes dual-stack support see [Dual-stack support with kubeadm](/docs/set
     ```sh
     docker run --rm -it \
     --net host \
-    -v /etc/kubernetes:/etc/kubernetes k8s.gcr.io/etcd:${ETCD_TAG} etcdctl \
+    -v /etc/kubernetes:/etc/kubernetes registry.k8s.io/etcd:${ETCD_TAG} etcdctl \
     --cert /etc/kubernetes/pki/etcd/peer.crt \
     --key /etc/kubernetes/pki/etcd/peer.key \
     --cacert /etc/kubernetes/pki/etcd/ca.crt \

```

**Comment by gaogao101** [comment]
@mengjiao-liu Thank you for your help. The latest changes have been submitted

**Review by Sea-n — COMMENTED** [review]
/approve

**Pull request merged**